README.md

```diff
@@ -113,29 +164,21 @@ When a user navigates to `[3]` trigerred incidents page.
 
 | Action                                                         | Key                           | Comment                                                                |
 |----------------------------------------------------------------|-------------------------------|------------------------------------------------------------------------|
-| Select/Deselect incident                                       | `Enter`⏎                      | Add or Remove an incident to be acknowledged.                          |
+| View Alerts for Incident                                       | `Enter`⏎                      | Lists all the alerts related to the incident. If there is a single alert, then it open ups the alert metadata                          |
 | Acknowledge incident(s)                                        | `ctrl-a`                      | Acknowledge the selected incidents.                                    |
 | Go back                                                        | `Esc`                         | Navigate back to alerts main view.                                     |
 
+### View Service Logs
+An alerting cluster's service logs can be viewed while viewing the alert data by pressing `L/l`.
 
-### View Incident Alerts
-
-To view alerts related to a particular incident, use the command:
+### View SOP for the alert
 
-```
-kite alerts <Incident ID>
-```
-This will list all the alerts belonging to that incident.
+To view SOP related to a particular alert a user can run these commands:
+* To select an alert and view its metadata press `enter`
+* To view SOP, press `S`
+This will open a new slide with the SOP. A user needs to make sure he/she has the access to `ops-sop` repository
```

Comment:
```suggestion
This will open a new slide with the SOP. A user needs to make sure he/she has the access to `ops-sop` repository
```
Can add link to the ops-sop repository